hadoop-ozone/ozone-manager/src/main/java/org/apache/hadoop/ozone/om/snapshot/SnapshotDiffManager.java

```diff
@@ -1155,21 +1157,16 @@ Set<String> getDeltaFiles(OmSnapshot fromSnapshot,
         LOG.warn("RocksDBCheckpointDiffer is not available, falling back to" +
                 " slow path");
       }
-
-      Set<String> fromSnapshotFiles =
-          RdbUtil.getSSTFilesForComparison(
-              ((RDBStore)fromSnapshot.getMetadataManager().getStore())
-                  .getDb().getManagedRocksDb(),
-              tablesToLookUp);
-      Set<String> toSnapshotFiles =
-          RdbUtil.getSSTFilesForComparison(
-              ((RDBStore)toSnapshot.getMetadataManager().getStore()).getDb()
-                  .getManagedRocksDb(),
-              tablesToLookUp);
+      ManagedRocksDB fromDB = ((RDBStore)fromSnapshot.getMetadataManager().getStore())
+          .getDb().getManagedRocksDb();
+      ManagedRocksDB toDB = ((RDBStore)fromSnapshot.getMetadataManager().getStore())
+          .getDb().getManagedRocksDb();
+      Set<String> fromSnapshotFiles = RdbUtil.getSSTFilesForComparison(fromDB, tablesToLookUp);
```

Comment:
Please use the helper function `SnapshotDiffManager#getSSTFileListForSnapshot()`;

Reply:
done